bigtop-manager-server/src/main/resources/application.yml

```diff
@@ -58,7 +58,7 @@ spring:
     show-sql: true
     properties:
       eclipselink:
-        ddl-generation: create-or-extend-tables
+        ddl-generation: create-tables
```

Comment:
I think this will still auto create tables

Reply:
setting it as 'none' doesn't insert into sequence table, I think i should add insert query in the ddl file like ambari did.

https://github.com/apache/ambari/blob/trunk/ambari-server/src/main/resources/Ambari-DDL-MySQL-CREATE.sql#L1159-L1218

After some test, I will correct the code. thank you